unit_testing/cli/test_cli_sct_detect_pmj.py

```diff
@@ -12,3 +14,9 @@ def test_sct_detect_pmj_backwards_compat(script_runner):
     logger.debug(f"{ret.stderr}")
     assert ret.success
     assert ret.stderr == ''
+
+def test_sct_detect_pmj_o_flag():
```

Comment:
Same comments re: subprocess and `tmp_path`.

Reply:
Tmp_path fixed. [main ](https://github.com/neuropoly/spinalcordtoolbox/blob/dc97d42730d7e603f83e54c68f242ad2bef7189b/spinalcordtoolbox/scripts/sct_detect_pmj.py#L260) does not take args.